packages/runner/src/main.js

```diff
@@ -1,14 +1,48 @@
 // Modules to control application life and create native browser window
-import {app, BrowserWindow, ipcMain} from 'electron'
+import {app, BrowserWindow, ipcMain ,Menu , remote} from 'electron'
 import {StudyWindow} from './study'
 
+import {proto, clock, test_stream} from './LSL_test'
+
 // Keep a global reference of the window object, if you don't, the window will
 // be closed automatically when the JavaScript object is garbage collected.
 let mainWindow
 
 // Check for development mode.
 const inDevelopment = process.env.NODE_ENV !== 'production'
 
+const template=[
```

Comment:
Nitpicking here, but if you have a moment, could you use a more descriptive variable name? Something like `menuEntries` or the like.